Bump GraphBLAS version to v4.x

szarnyasg opened the pull request.

szarnyasg committed:
Bump GraphBLAS version to v4.x

szarnyasg committed:
Migration to GrBv4 WIP

szarnyasg commented:
Pending issue: https://github.com/GraphBLAS/LAGraph/pull/110
There's also one compilation bug around the wrapper that I could not figure out yet.

marci543 committed:
Import gb_utils updates

Based on ldbc/sigmod2014-contest-graphblas@0618fc9870e6e96480cf1915203b71adef5224a3

marci543 committed:
Add ok() for LAGraph errors

marci543 committed:
Use LAGraph_Finalize

marci543 committed:
Use LAGraph ok()

marci543 committed:
Add LAGraph_Graph, to_LAGraph to gb_utils

marci543 committed:
Use LAGraph function

marci543 committed:
Fix include

marci543 committed:
Fix threading

marci543 committed:
WIP! TODO! Use temporary GraphBLAS base image, only run test this

Dockerfile available from: marci543/docker-graphblas@4c7f89f3b776714288fa866fff58f1e322c4c5ba

marci543 commented:
It compiles now, based on a temporary Docker image.

Needed:
- [ ] bump [Docker image](https://github.com/bergkvist/docker-graphblas) for GraphBLAS v4.0.3
- [ ] finalizing [reorg branch of LAGraph](https://github.com/GraphBLAS/LAGraph/tree/reorg)
- [ ] release [LAGraph image](https://github.com/bergkvist/docker-graphblas)
- [ ] remove temporary commit: 126bcd9173166ab91fceb04c3b7a51639c73eb26
- [x] update build instructions in readme

marci543 committed:
WIP! TODO! Use old base image

marci543 committed:
Reorder Post and Comment load

marci543 committed:
Build and check root_post from commented in Q1 initial

marci543 committed:
Use debug mode when tests are running

marci543 committed:
Build and check root_post from commented in Q1 updates

marci543 committed:
Inline Q1 update details

marci543 committed:
Remove reading root_post edges

marci543 committed:
Rename

marci543 committed:
Use new common base image with new data

This reverts commit 3ef47946 "WIP! TODO! Use old base image"

marci543 committed:
Bump install instructions

marci543 committed:
Test all tools in CI

This partially reverts commit "WIP! TODO! Use temporary GraphBLAS base image, only run test this" 126bcd9173166ab91fceb04c3b7a51639c73eb26

marci543 committed:
Merge remote-tracking branch 'origin/master' into graphblas-v4

marci543 committed:
Build GraphBLAS #87
